src/cloud/release-notes/backward-incompatible-changes.md

```diff
@@ -1,37 +1,36 @@
 ---
-group: release-notes
-title: Backward Incompatible Changes
+group: cloud-guide
+title: Backward incompatible changes
 ---
 
-Use the following information to learn about backward incompatible changes that require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
+Use the following information to learn about backward incompatible changes that might require you to adjust Cloud configuration and processes for existing Cloud projects when you upgrade to the latest release of the {{site.data.var.ct}} package or other {{site.data.var.csuite}} packages.
 
 ### Service version requirement changes
 
-**PHP minimum version requirement**–In {{ site.data.var.ct }} v2002.1.0 and later, the minimum php version requirement is 7.1.3+. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file. 
+We changed the minimum PHP version requirement from 7.1.0 to 7.1.3+ for Cloud projects that use `{{ site.data.var.ct }}` v2002.1.0 and later. If your environment configuration specifies an earlier version, update the [php configuration]({{ site.baseurl }}/cloud/project/project-conf-files_magento-app.html#configure-php-options) in the `.magento.app.yaml` file.
```

Comment:
In `.magento.app.yaml` you still keep it as 7.1 (you do not specify patch versions). Also 2002.0.x supported PHP 7.x - this is the main change. We need to change something in config only if we used PHP 7.x. I would just have it as `from 7.0.x to 7.1.x`.